scripts/install-build-deps

```diff
@@ -25,7 +25,6 @@ readonly SELF="$(readlink -f $0)"
 readonly TOP_SRCDIR="${SELF%/*/*}"
 dry_run=
 install_epel=true
-install_isal=true
```

Comment:
Is this not required?

Reply:
No. It was added to add isa-l in skip_tags in case user wants to skip Intel ISA library installation. But now it is mandatory to install it, we don't need this flag as we install the library by default.